libs/extensions-api/registry/extension.registry.ts

```diff
@@ -13,7 +15,7 @@ export class UmbExtensionRegistry {
 	private _extensions = new BehaviorSubject<Array<ManifestBase>>([]);
 	public readonly extensions = this._extensions.asObservable();
 
-	register(manifest: ManifestTypes): void {
+	register(manifest: ManifestTypes, rootHost?: UmbControllerHostInterface): void {
```

Comment:
We should probably make the host a part of the constructor, but we can change that when we provide it trough the context API instead.